src/components/pages/login/form.tsx

```diff
@@ -81,7 +81,11 @@ function LoginForm({ onSuccess, redirect = true, forward }: ISignInFormProps) {
       }
 
       // hard redirect to re-build cache
-      window.location.assign("/");
+      if (forward && redirect) {
```

Comment:
can you use ternary operator here?

Reply:
it also uses async and await so ternary is not recommended.